OBSDOCS-794: Add known Splunk log forwarding issue to 5.8 release notes (#73122)
openshift/openshift-docs: modules/logging-release-notes-5-8-0.adoc

[id="logging-release-notes-5-8-0-known-issues"]
== Known Issues

> * Currently, Splunk log forwarding might not work after upgrading to version 5.8 of the {clo}. This issue was caused by transitioning from OpenSSL version 1.1.1 to version 3.0.7. In the newer OpenSSL version, there is a default behavior change, where connections to TLS 1.2 endpoints are rejected if they do not expose the link:https://datatracker.ietf.org/doc/html/rfc5746[RFC 5746] extension.

Reviewer: ```suggestion
* Currently, Splunk log forwarding might not work after upgrading to version 5.8 of the {clo}. This issue is caused by transitioning from OpenSSL version 1.1.1 to version 3.0.7. In the newer OpenSSL version, there is a default behavior change, where connections to TLS 1.2 endpoints are rejected if they do not expose the link:https://datatracker.ietf.org/doc/html/rfc5746[RFC 5746] extension.
```
I would keep present tense here to avoid any confusion that the issue is resolved.

+

> The workaround for this issue requires users to enable TLS 1.3 support on the TLS terminating load balancer in front of the Splunk HEC (HTTP Event Collector) endpoint. Splunk is a third-party system and this should be configured from the Splunk end.

Reviewer: ```suggestion
As a workaround, enable TLS 1.3 support on the TLS terminating load balancer in front of the Splunk HEC (HTTP Event Collector) endpoint. Splunk is a third-party system and this should be configured from the Splunk end.
```
